### ISSUE TITLE
email rejections

### ISSUE THREAD

**Issue by scd** [theCrag/website#2160]
We are getting a large percentage of email rejections in SparkPost from google because of the following:

 550 5.7.1 Unconfigured Sending Domain <gmail.com> 	gmail.com 	Policy Rejection 	24 

As far as I can see it is all set up correctly.

**Comment by scd** [comment]
I wonder if I have to set up an additional email tag for the sender as opposed to the from address?

**Comment by brendanheywood** [comment]
http://stackoverflow.com/questions/34961429/sparkpost-rails-sending-domain-rejected ?

**Comment by scd** [comment]
It looks like I have to set the helo parameter which is supposed to say who you are

**Comment by scd** [comment]
All the sparkpost setup looks right. I just logged into sparkpost again, and the delivery stats seem much better this week.

Anyway I have added the helo parameter, so lets see how it goes.

http://search.cpan.org/~fayland/Email-Sender-Transport-SMTP-TLS-0.15/lib/Email/Sender/Transport/SMTP/TLS.pm

http://www.perlmonks.org/?node_id=845523

This may be totally unrelated, so I think we need to monitor for a while
